api/reviews_api.py

```diff
@@ -160,8 +160,6 @@ def do_patch(self, **kwargs) -> dict[str, str]:
     new_assignees = request.assignees
     new_answers = request.survey_answers
 
-    logging.info('@@@ request is %r', request)
```

Comment:
Is this meant to be removed?

Reply:
Yeah, that was left-over debugging that is not needed.